addons-l10n/en/variable-manager.json

```diff
@@ -2,5 +2,6 @@
   "variable-manager/for-this-sprite": "Variables for this sprite",
   "variable-manager/for-all-sprites": "Variables for all sprites",
   "variable-manager/variables": "Variables",
-  "variable-manager/search": "Search"
+  "variable-manager/search": "Search",
+  "variable-manager/too-big": "Click to display very large value."
```

Comment:
Could this message state the max length? eg. `Click to display very large value. (1000000+ characters)` (with a separate message for lists) or would that be unnecessary information?

Reply:
I don't think it's really necessary to tell users what the exact limit is.